<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulationWindow</class>
 <widget class="QMainWindow" name="SimulationWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NetSim - The educational Network Simulation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../assets/ressources.qrc">
    <normaloff>:/icon.svg</normaloff>:/icon.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="networkTab">
       <attribute name="icon">
        <iconset resource="../../../assets/ressources.qrc">
         <normaloff>:/network.svg</normaloff>:/network.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Network</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="packageTab">
       <attribute name="icon">
        <iconset resource="../../../assets/ressources.qrc">
         <normaloff>:/package.svg</normaloff>:/package.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Packages</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableView" name="packagesTableView">
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="sourceTab">
       <attribute name="icon">
        <iconset resource="../../../assets/ressources.qrc">
         <normaloff>:/source.svg</normaloff>:/source.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Sources</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="processTab">
       <attribute name="icon">
        <iconset resource="../../../assets/ressources.qrc">
         <normaloff>:/process.svg</normaloff>:/process.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Processes</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentation"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuSimulation"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../assets/ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>SimulationWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>959</x>
     <y>539</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
